cpp/src/arrow/engine/substrait/serde_test.cc

```diff
@@ -411,21 +419,18 @@ TEST(Substrait, CannotDeserializeLiteral) {
 }
 
 TEST(Substrait, FieldRefRoundTrip) {
-  for (FieldRef ref : {
-           // by name
-           FieldRef("i32"),
-           FieldRef("ts_ns"),
-           FieldRef("struct_i32_str"),
-
-           // by index
-           FieldRef(0),
-           FieldRef(1),
-           FieldRef(kBoringSchema->num_fields() - 1),
-           FieldRef(kBoringSchema->GetFieldIndex("struct_i32_str")),
-
-           // nested
-           FieldRef("struct_i32_str", "i32"),
-           FieldRef(kBoringSchema->GetFieldIndex("struct_i32_str"), 1),
+  for (FieldRef ref :
+       {// by name
+        FieldRef("i32"), FieldRef("ts_ns"), FieldRef("struct"),
+
+        // by index
+        FieldRef(0), FieldRef(1), FieldRef(kBoringSchema->num_fields() - 1),
+        FieldRef(kBoringSchema->GetFieldIndex("struct")),
+
+        // nested
+        FieldRef("struct", "i32"), FieldRef("struct", "struct_i32_str", "i32"),
+        FieldRef(kBoringSchema->GetFieldIndex("struct"), 1)
```

Comment:
nit: The trailing comma causes clang-format to put each of the elements on it's own line
```suggestion
        FieldRef(kBoringSchema->GetFieldIndex("struct"), 1),
```

Reply:
Good to know; I was wondering why it was messing it up. 45dd533